awx/main/migrations/0062_v350_credential_plugins.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import taggit.managers
+
+# AWX
+import awx.main.fields
+from awx.main.models import CredentialType
+
+
+def setup_tower_managed_defaults(apps, schema_editor):
+    CredentialType.setup_tower_managed_defaults()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
I want to revisit two lines these before we merge and figure out why Django is auto-adding them.

Reply:
I tried recreating these by hand, and Django insists on adding them when we run `makemigrations`.  @matburt do you have enough Django-fu to guess why 🤷‍♂️ ?

Reply:
![kungfu](https://user-images.githubusercontent.com/9753817/53578660-0e966e80-3b46-11e9-9c49-e7a70d4380c4.gif)